docs/content/guides/cell-types/handsontable-cell-type.md

```diff
@@ -11,11 +11,9 @@ searchCategory: Guides
 
 # Handsontable cell type
 
-[[toc]]
-
-## Overview
+Use the Handsontable cell type to collect user input with an interactive spreadsheet editor in a popup.
```

Comment:
@kirszenbaum 

> Use the Handsontable cell type to collect user input with an interactive spreadsheet editor in a popup.

The Handsontable cell type is a solid table that appears when we open cell editor. I'm not sure if we can call it a `user input`, and the `interactive spreadsheet editor` sounds like we can alter the data of the table bit we can't.